database/schema.sql

```diff
@@ -425,7 +426,8 @@ INSERT INTO `awfy_vendor` (`id`, `name`, `vendor`, `csetURL`, `browser`, `rangeU
 (11, 'SpiderMonkey', '1GB inbound', 'http://hg.mozilla.org/integration/mozilla-inbound/rev/', 'Firefox OS', 'http://hg.mozilla.org/integration/mozilla-inbound/pushloghtml?fromchange={from}&tochange={to}'),
 (12, 'SpiderMonkey', '512MB inbound', 'http://hg.mozilla.org/integration/mozilla-inbound/rev/', 'Firefox OS', 'http://hg.mozilla.org/integration/mozilla-inbound/pushloghtml?fromchange={from}&tochange={to}'),
 (13, 'SpiderMonkey', '256MB inbound', 'http://hg.mozilla.org/integration/mozilla-inbound/rev/', 'Firefox OS', 'http://hg.mozilla.org/integration/mozilla-inbound/pushloghtml?fromchange={from}&tochange={to}'),
-(14, 'V8', 'Google', 'https://chromium.googlesource.com/v8/v8/+/', 'Chrome', 'http://code.google.com/p/v8/source/list?num={num}&start={to}');
+(14, 'V8', 'Google', 'https://chromium.googlesource.com/v8/v8/+/', 'Chrome', 'http://code.google.com/p/v8/source/list?num={num}&start={to}'),
+(15, 'SpiderMonkey', 'Mozilla', 'https://github.com/servo/servo/commits', 'Servo', 'https://github.com/servo/servo/compare/{from}...{to}');
 
```

Comment:
I would prefer to do this using the migration scripts now.
-> insert vendor, retrieve ID
-> insert mode with right vendor_id

migration scripts are quite easy. Just a php file containing the function migrate(), which should migrate to the new version. And rollback() which should undo the migrate() function. Like:
https://github.com/h4writer/arewefastyet/blob/demotestamptosortorder/database/migration-1.php